pkg/scheduler/framework/plugins/noderesources/fit_test.go

```diff
@@ -534,25 +534,25 @@ func TestNotEnoughRequests(t *testing.T) {
 		{
 			pod:        &v1.Pod{},
 			nodeInfo:   framework.NewNodeInfo(newResourcePod(framework.Resource{MilliCPU: 10, Memory: 20})),
-			name:       "even without specified resources predicate fails when there's no space for additional pod",
+			name:       "even without specified resources, predicate fails when there's no space for additional pod",
 			wantStatus: framework.NewStatus(framework.Unschedulable, "Too many pods"),
 		},
 		{
 			pod:        newResourcePod(framework.Resource{MilliCPU: 1, Memory: 1}),
 			nodeInfo:   framework.NewNodeInfo(newResourcePod(framework.Resource{MilliCPU: 5, Memory: 5})),
-			name:       "even if both resources fit predicate fails when there's no space for additional pod",
+			name:       "even if both resources fit, predicate fails when there's no space for additional pod",
 			wantStatus: framework.NewStatus(framework.Unschedulable, "Too many pods"),
 		},
 		{
 			pod:        newResourcePod(framework.Resource{MilliCPU: 5, Memory: 1}),
 			nodeInfo:   framework.NewNodeInfo(newResourcePod(framework.Resource{MilliCPU: 5, Memory: 19})),
-			name:       "even for equal edge case predicate fails when there's no space for additional pod",
+			name:       "even for equal edge case, predicate fails when there's no space for additional pod",
 			wantStatus: framework.NewStatus(framework.Unschedulable, "Too many pods"),
 		},
 		{
 			pod:        newResourceInitPod(newResourcePod(framework.Resource{MilliCPU: 5, Memory: 1}), framework.Resource{MilliCPU: 5, Memory: 1}),
 			nodeInfo:   framework.NewNodeInfo(newResourcePod(framework.Resource{MilliCPU: 5, Memory: 19})),
-			name:       "even for equal edge case predicate fails when there's no space for additional pod due to init container",
+			name:       "even for equal edge case for init containers, predicate fails when there's no space for additional pod",
```

Comment:
```suggestion
			name:       "even for equal edge case, predicate fails when there's no space for additional pod due to  init container",
```

This is actually more clear to me.

Reply:
Thanks for your feedback :)

https://github.com/kubernetes/kubernetes/blob/master/pkg/scheduler/framework/plugins/noderesources/fit_test.go#L561
https://github.com/kubernetes/kubernetes/blob/master/pkg/scheduler/framework/plugins/noderesources/fit.go#L257

It seems to fail because the number pod({MilliCPU: 5, Memory: 19}) inside node and + 1 is greater than AllowedPodNumber(1). 1 +1 > 1
I think init container is not related with this one, so I changed the line.
Correct me if I am wrong.

Reply:
We construct the pod with `newResourceInitPod` here, so we want to highlight the init container, then it matters. Anyway, I think it doesn't make a lot of sense to polish this comments.

Reply:
Thanks for your nice explanation. I will leave as it is.